lib/dor/services/embargo_service.rb

```diff
@@ -55,6 +56,7 @@ def release_20_pct_vis(release_agent)
 
       events.add_event('embargo', release_agent, '20% Visibility Embargo released')
     end
+    deprecation_deprecate release_20_pct_vis: 'this moved to dor-service-app'
 
     def update(new_date)
```

Comment:
was lack of deprecation warning on this method intentional?

Reply:
yeah - did you put the deprecation warning in the wrong place?  Or just omit one?

Reply:
`#update` is not being deprecated